TS3Client/Generated/Messages.cs

```diff
@@ -9467,6 +9467,8 @@ public enum NotificationType
 		VersionRequest,
 		///<summary>[C2S] ntfy:whoami</summary>
 		WhoAmIRequest,
+		///<summary>[S2C] ntfy:any</summary>
+		RawCommand,
```

Comment:
You do realize that's not how it works. Those are autogenerated functions. And just adding a `RawCommand` will not work. If you that desperately need a raw command, I can see what I can do there.

Reply:
Yes, please. for simplicity sake